src/styles/styles.css

```
@@ -0,0 +1,121 @@
+* {
+  box-sizing: border-box;
+}
+
+html {
+  font-size: 62.5%;
+  font-family: Arial, Helvetica, sans-serif;
+}
+
+.wrapper {
+  margin-right: auto;
+  margin-left: auto;
+  width: 60%;
+}
+
+header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  border-bottom: solid 1px #e8d2d2;
+  padding: 1.8rem 0;
+}
+
+.header__slogan {
+  color: #827b7b;
+  text-transform: uppercase;
+  font-weight: 100;
+  font-size: 1.2rem;
+  letter-spacing: 1px;
+  margin: 0 2rem;
+  text-align: center;
+}
+
+.filters {
+  display: grid;
+  gap: 2rem;
+  margin: 2rem auto 3rem;
+}
+
+.input {
+  background: transparent;
+  border: solid 2px #e8d2d2;
+  border-radius: 5px;
+  height: 3rem;
+  font-size: 1.5rem;
+  padding: 0 0.8rem;
+  color: #827b7b;
+  outline: none;
+}
+
+.grid-auto-fill {
+  display: grid;
+  grid-template-columns: repeat(auto-fill, minmax(24rem, 1fr));
+  gap: 2.8rem;
+}
+
+.card {
+  border: solid 1px #f9f2f2;
```

Ajustar a ordem dos parâmetros:

```suggestion
  border: 1px solid #f9f2f2;
```